packages/opentelemetry-sdk-trace-base/test/common/BasicTracerProvider.test.ts

```diff
@@ -90,6 +91,29 @@ describe('BasicTracerProvider', () => {
         const tracer = new BasicTracerProvider();
         assert.ok(tracer.activeSpanProcessor instanceof NoopSpanProcessor);
       });
+      it('should use noop span processor by default and no diag error', () => {
+        const errorStub = sinon.spy(diag, 'error');
+        const tracer = new BasicTracerProvider();
+        assert.ok(tracer.activeSpanProcessor instanceof NoopSpanProcessor);
+
+        sinon.assert.notCalled(errorStub);
+      });
+    });
+
+    describe('when user sets unavailable exporter', () => {
+      it('should use noop span processor by default and show diag error', () => {
+        const errorStub = sinon.spy(diag, 'error');
+        env.OTEL_TRACES_EXPORTER = 'someExporter';
+
+        const tracer = new BasicTracerProvider();
+        assert.ok(tracer.activeSpanProcessor instanceof NoopSpanProcessor);
+
+        sinon.assert.calledWith(
+          errorStub,
+          'Exporter "someExporter" requested through environment variable is unavailable.'
+        );
+        delete env.OTEL_TRACES_EXPORTER;
+      });
```

Comment:
Looks like moving this test to `sdk-trace-node` and the `NodeTracerProvider` tests would do the trick of fixing the Browser tests, as the code in `sdk-trace-base` is intended for both node and the browser. :slightly_smiling_face:

Reply:
It looks like changing that test to use `envSource` instead of `env` directly fixed the browser tests. I just had to use the `envSource` that was [defined here](https://github.com/open-telemetry/opentelemetry-js/blob/947b3073ed6723be1b202ec76c6784fa7b0acd36/packages/opentelemetry-sdk-trace-base/test/common/BasicTracerProvider.test.ts#L64-L68).

Reply:
Ah, nice :slightly_smiling_face: 
Looks like I missed that one, that's a better fix - thanks! :slightly_smiling_face: